README.md

```diff
@@ -87,8 +87,8 @@ libraryDependencies += "com.github.VirtusLab.akka-serialization-helper" %% "bore
-| Boilerplate | a lot: <ul><li>ADTs requires amount of annotation equal to or exceeding the actual type definitions</li><li>poor support for case object - requires custom serializer and deserializer and sealed trait per case object, otherwise, it creates new singletons and breaks pattern matching</ul> | a bit: every top level sealed trait must be mentioned for Circe to generate a codec | in case of custom types a second layer of models is needed | sometimes requires annotations | considerable: every top level sealed trait must have manually defined codec | none
+| Schema evolution | <ul><li>removing field</li><li>adding optional field</li></ul> with [`JacksonMigration`](https://doc.akka.io/docs/akka/current/serialization-jackson.html#schema-evolution) <ul><li>adding mandatory field</li><li>renaming field</li><li>renaming class</li><li>support of forward versioning for rolling updates</li></ul>| <ul><li>adding optional field</li><li>removing optional field</li><li>adding required field with default value</li><li>removing required field</li><li>renaming field</li><li>reordering fields</li><li>transforming data before deserialization</li></ul> | <ul><li>adding optional field</li><li>removing optional field</li><li>adding required field with default value</li><li>removing required field</li><li>renaming field</li><li>reordering fields</li><li>changing between compatible types</li></ul>  | <ul><li>reordering fields</li><li>renaming fields</li><li>adding optional field</li><li>adding required field with default value</li><li>removing field with default value</li></ul> | <ul><li>renaming fields</li><li>transforming data before deserialization</li></ul> | <ul><li>adding field</li><li>removing field</li><li>renaming field</li><li>renaming class</li></ul>
+| Boilerplate | a lot: <ul><li>ADTs requires amount of annotation equal to or exceeding the actual type definitions</li><li>requires custom serializer and deserializer and sealed trait per case object</ul> | every top level sealed trait must be mentioned for Circe to generate a codec | in case of custom types a second layer of models is needed | sometimes requires annotations | every top level sealed trait must have manually defined codec | every class must be passed to register method with unique id
```

Comment:
Nit:

```suggestion
| Boilerplate | a lot: <ul><li>ADTs requires amount of annotation equal to or exceeding the actual type definitions</li><li>requires custom serializer and deserializer and sealed trait per case object</ul> | every top level sealed trait must be mentioned for Circe to generate a codec | in case of custom types a second layer of models is needed | sometimes requires annotations | every top level sealed trait must have manually defined codec | every class must be passed to `register` method with unique id
```

I suppose that's what you meant (?)